app/public/js/noTimes.js

```diff
@@ -0,0 +1,7 @@
+(function (global) {
+  'use strict';
+  var $ = global.jQuery;
+  $('p.callout--info__mesg:contains("We can\'t find any opening times")').each(function(){
```

Comment:
Mmh, this isn't great. Although I can't find a better way to do it. Can the string be a little less strict, something like just 'find any opening times'? The message comes from the API so it is going to be difficult to detect changes.
@markysoft any ideas?

Reply:
I think that message occurs when there are no opening times on the services object, so you could check if the service.openingTimes is undefined (or closed all week?) and add a 'noOpeningTimes' property to the service

Reply:
In fact, I don't think this needs to be done on the client at all.